public/js/reports.js

```diff
@@ -0,0 +1,13 @@
+const createButton = document.getElementById('create');
+const date = document.getElementById('month');
+createButton.addEventListener('click', (e) => {
```

Comment:
use descriptive variables names for more readability
you can use `element` or `item` etc...